### ISSUE TITLE
Uncaught ReferenceError: openCategory is not defined

### ISSUE THREAD

**Issue by PkChris** [sindrets/atom-title-bar-replacer#9]
[Enter steps to reproduce:]

1. Open Atom
2. Error appears

Just got this today after updating atom-title-bar-replacer

**Atom**: 1.19.6 x64
**Electron**: 1.6.9
**OS**: Microsoft Windows 10 Pro
**Thrown From**: [title-bar-replacer](https://github.com/sindrets/atom-title-bar-replacer) package 1.4.0

### Stack Trace

Uncaught ReferenceError: openCategory is not defined

```
At C:\Users\PkChris\.atom\packages\title-bar-replacer\lib\title-bar-replacer-view.js:256

ReferenceError: openCategory is not defined
    at /packages/title-bar-replacer/lib/title-bar-replacer-view.js:256:77)
    at HTMLDivElement.dispatch (/packages/title-bar-replacer/node_modules/jquery/dist/jquery.js:4737:27)
    at HTMLDivElement.elemData.handle (/packages/title-bar-replacer/node_modules/jquery/dist/jquery.js:4549:28)
```

### Commands

```
     -0:13.9.0 application:show-settings (atom-text-editor.editor.is-focused)
```

### Non-Core Packages

```
title-bar-replacer 1.4.0 
```

**Comment by PkChris** [comment]
Hmm, it seems to have fixed itself after I reset the package settings to default. Weird.

**Comment by PkChris** [comment]
Started happening again on restart of Atom.

**Comment by sindrets** [comment]
@PkChris Yes, you're absolutely right. I restructured a bunch of code for 1.4 and that variable didn't come along in the process. I published a hotfix 1.4.1.

**Comment by PkChris** [comment]
Oh I know how that is, I've been restructuring a lot of code lately running into the same kind of issues. Thanks for the quick fix! I love this package, makes the UI blend in a lot nicer on Windows.
